build.gradle

```
@@ -47,20 +43,44 @@ jar {
 
 dependencies {
     api platform('com.exactpro.th2:bom:4.0.2')
-    implementation 'com.exactpro.th2:common:5.0.0-dev-version-5-3838510969-SNAPSHOT'
-    implementation 'io.reactivex.rxjava3:rxkotlin:3.0.1'
-
     implementation 'org.slf4j:slf4j-log4j12'
     implementation 'org.slf4j:slf4j-api'
 
+    implementation 'com.exactpro.th2:common:5.0.0-dev-version-5-3838510969-SNAPSHOT'
+
+    implementation 'io.reactivex.rxjava3:rxkotlin:3.0.1'
     implementation 'org.apache.kafka:kafka_2.13:3.3.1'
     implementation 'io.github.microutils:kotlin-logging:3.0.4'
 
     testImplementation 'org.junit.jupiter:junit-jupiter:5.7.2'
 }
 
+test {
+    useJUnitPlatform()
+}
+
+applicationName = 'service'
+
 application {
     mainClassName 'com.exactpro.th2.kafka.client.Main'
 }
 
-applicationName = 'service'
\ No newline at end of file
+distTar {
+    archiveName "${applicationName}.tar"
+}
+
+dockerPrepare {
+    dependsOn distTar
+}
+
+docker {
+    copySpec.from(tarTree("$buildDir/distributions/${applicationName}.tar"))
+}
+
+compileKotlin {
+    kotlinOptions.jvmTarget = "11"
+}
+
+compileTestKotlin {
+    kotlinOptions.jvmTarget = "11"
+}
```

You can use this instead
```suggestion
tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile) {
    kotlinOptions {
        jvmTarget = "11"
    }
}
```